nexus_system_orchestrator/src/system_orchestrator.cpp

```diff
@@ -725,7 +725,7 @@ _parse_wo(const std::string& work_order_id, const common::WorkOrder& work_order)
       work_order.metadata().has_value() ? work_order.metadata().value() :
       YAML::Node{};
     YAML::Emitter out;
-    out << processed_step;
+    out << step.yaml;
```

Comment:
We actually want to keep the original here as this is how embed the metadata that is part of the WorkOrder (above the list of steps) in the WorkcellTask for a workcell. Without this, the workcell only receives information defined within a `Step` but there could be globally relevant information in the WorkOrder that we would like to pass along to each workcell as well. 

```suggestion
    out << processed_step;
```

Reply:
Makes sense, thanks for catching that, https://github.com/osrf/nexus/pull/87/commits/3ff960d2e8b7751f55b025747856d883cf653a53